### ISSUE TITLE
Interface Is not Accessible With Screen Readers

### ISSUE THREAD

**Issue by jsl303** [rncbc/qjackctl#98]
None of the UI elements are accessible with screen readers.
You could test with either Narrator on Windows 10 (>= Fall Creators Update v1709), or VoiceOver on MacOS.
https://doc.qt.io/qt-5/accessible.html

**Comment by HaHeho** [comment]
If required, isn't a fallback to the command line tools in order to administrate JACK the more accessible solution than trying to make a GUI work?

**Comment by jsl303** [comment]
Can you manage routing in command line to connect and disconnect ports?

**Comment by HaHeho** [comment]
There is a multitude of other executable that come with an installation or are build. However, I didn't really find a good place where they are documented.
https://github.com/jackaudio/jackaudio.github.com/wiki#jack-tools

For your purpose `jack_lsp` (list ports) and `jack_connect` (connect ports) are the tools of choice. Curiously, neither is contained in the list above. No idea what is going on there. On Windows it might look like this:
```
C:\Users\hanne>jack_lsp
system:capture_1
system:capture_2
system:capture_3
system:capture_4
system:capture_5
system:capture_6
system:capture_7
system:capture_8
system:capture_9
system:capture_10
system:capture_11
system:capture_12
system:capture_13
system:capture_14
system:capture_15
system:capture_16
system:capture_17
system:capture_18
system:playback_1
system:playback_2
system:playback_3
system:playback_4
system:playback_5
system:playback_6
system:playback_7
system:playback_8
system:playback_9
system:playback_10
system:playback_11
system:playback_12
system:playback_13
system:playback_14
system:playback_15
system:playback_16
system:playback_17
system:playback_18

C:\Users\hanne>jack_connect

usage: jack_connect [options] port1 port2
Connects two JACK ports together.

        -s, --server <name>   Connect to the jack server named <name>
        -v, --version         Output version information and exit
        -h, --help            Display this help message

For more information see http://jackaudio.org/

C:\Users\hanne>jack_connect system:capture_1 system:playback_1

```

**Comment by jsl303** [comment]
I just compiled with qt 5.15, and many elements on QJackCtl seem to be 
accessible with VoiceOver. Unfortunately can't seem to access routing 
window.
Thanks for letting me know about those command line tools. I'm on Mac, 
many of them listed on the page are not in my system for some reason.
At least jack_connect, jack_disconnect, jack_lsp seem to be installed.

**Comment by jsl303** [comment]
Is there a command line tool that shows all the existing connections?

**Comment by jsl303** [comment]
Sorry, it looks like jack_lsp -c does that.
